desktop/troubleshoot/topics.md

```diff
@@ -261,16 +261,17 @@ To install Hyper-V manually, see [Install Hyper-V on Windows 10](https://msdn.mi
 From the start menu, type **Turn Windows features on or off** and press enter.
 In the subsequent screen, verify that Hyper-V is enabled.
 
-#### Virtualization must be enabled
+#### Nested virtualization must be enabled
```

Comment:
I **think** the term nested virtualization only applies to the VDI context, right? Virtualization must be enabled is still correct, however if you're running DD in a VDI environment, you must also ensure that it supports nested virtualization, and that nested virtualization is enabled ([ex][1])

(I could be wrong here -- just thinking out loud, would be good with a 2nd opinion)

[1]: https://learn.microsoft.com/en-us/virtualization/hyper-v-on-windows/user-guide/nested-virtualization

Reply:
You are indeed correct. I went back to support to confirm as well. Have amended it accordingly - thank you for catching this. VM stuff makes my brain warm to the touch 🤯

Reply:
very much the same for me!